tests/scene/test_bit_map.h

```diff
@@ -349,6 +349,49 @@ TEST_CASE("[BitMap] Grow and shrink mask") {
 	CHECK_MESSAGE(bit_map.get_bitv(Point2i(131, 131)) == false, "Bits that are on the edge of the shape should be set to false");
 }
 
+TEST_CASE("[BitMap] Grow and shrink very large mask") {
+	const Size2i dim{ 2040, 2040 };
+	BitMap bit_map{};
```

Comment:
```suggestion
	Ref<BitMap> bit_map;
	bit_map.instantiate();
```
And use pointer access instead, as this is a `RefCounted` class